int64 nodeId
geometry_msgs/PoseStamped pose